docs/news.rst

```diff
@@ -3,6 +3,92 @@
 Release notes
 =============
 
+.. _release-2.11.2:
+
+Scrapy 2.11.2 (2024-05-DD)
```

Comment:
The day will be added when we create the release tag?

Reply:
Yes. Hopefully today.